sdk/mx.sdk/mx_sdk_vm.py

```diff
@@ -106,6 +106,7 @@ def __init__(self, destination, jar_distributions, build_args, use_modules=None,
         self.jar_distributions = jar_distributions
         self.build_args = build_args
         self.use_modules = use_modules
+        self.main_module = main_module
```

Comment:
Can a library have a "main module"? or should this rather be an attribute of `LauncherConfig`?

Reply:
While currently not implemented we could use that to only search for `@CEntryPoint`s in the specified module (if specified)

Reply:
But you are right that it is better for now not to have it on `AbstractNativeImageConfig` level.

Reply:
Fixed in https://github.com/oracle/graal/pull/3446/commits/91941f9adf5f9480491a2c07c7a1304008f4788f